core/dbt/include/global_project/macros/python_model/python.sql

```diff
@@ -3,7 +3,7 @@
     {%- set ref_dict = {} -%}
     {%- for _ref in model.refs -%}
         {%- set resolved = ref(*_ref) -%}
-        {%- do ref_dict.update({_ref | join("."): resolved.quote(database=False, schema=False, identifier=False) | string}) -%}
+        {%- do ref_dict.update({_ref | join("."): resolved | string | replace('"', '\"')}) -%}
```

Comment:
is the `"` here tied to specific quoting for snowflake?

Reply:
I believe it would be relevant to any data platform that uses `"` as its quoting character ([it's our default](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/adapters/base/relation.py#L32)), or any identifier that contains a quote character (`"`) for any reason